### PR TITLE
RAS-939 Survey Status Change

### PR DESCRIPTION
**Pull request by arroyoAle**
# What and why?
Adds a new case group transition from `complete` to `not started` to allow business users a new status change in rOps
# How to test?
- deploy this and https://github.com/ONSdigital/response-operations-ui/pull/914 to your environment
- run the acceptance tests to populate your env
- log into frontstage and complete the survey
- go to http://localhost:8085/reporting-units/49900000001 and navigate to change the status from complete to not started
- check that the survey appears in the todo page in frontstage
# Jira
https://jira.ons.gov.uk/browse/RAS-939

### CODE REVIEW

File: src/test/java/uk/gov/ons/ctp/response/casesvc/state/StateTransitionManagerUnitTest.java [hunk 1]
```diff
@@ -580,4 +580,18 @@ public void givenUnknownEligibilityWhenEqLaunchThenComplete() throws CTPExceptio
...
+    // Given
+    CaseGroupStatus complete = CaseGroupStatus.COMPLETE;
+
+    // When
+    CaseGroupStatus destinationState =
+        caseGroupStateMachine.transition(
+            complete, CategoryDTO.CategoryName.OFFLINE_RESPONSE_PROCESSED);
+
+    // Then
+    assertEquals(CaseGroupStatus.COMPLETE, destinationState);
```
Comment (LJBabbage) [inline src/test/java/uk/gov/ons/ctp/response/casesvc/state/StateTransitionManagerUnitTest.java:595]: Is this right? I was expecting not started

Comment (arroyoAle) [reply src/test/java/uk/gov/ons/ctp/response/casesvc/state/StateTransitionManagerUnitTest.java:595]: You're right I've fixed this

**Review by SteveScorfield — APPROVED** [review]
Looks good to me. Tested and reviewed.

File: src/main/java/uk/gov/ons/ctp/response/casesvc/state/CaseSvcStateTransitionManagerFactory.java [hunk 2]
```diff
@@ -262,6 +262,10 @@ private StateTransitionManager<CaseState, CaseEvent> caseStateTransitionManager(
         CaseGroupStatus.NOLONGERREQUIRED,
         CategoryDTO.CategoryName.COMPLETED_TO_NOTSTARTED,
         CaseGroupStatus.NOTSTARTED);
+    builder.put(
```
[the remark this answers is not in the document]
Comment (arroyoAle) [reply src/main/java/uk/gov/ons/ctp/response/casesvc/state/CaseSvcStateTransitionManagerFactory.java:265]: Comment has been updated

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**